okhttp/src/test/java/io/grpc/okhttp/OkHttpClientTransportTest.java

```diff
@@ -144,12 +145,14 @@ public void setUp() {
     frameReader = new MockFrameReader();
   }
 
-  private void initTransport() {
-    initTransport(3, new ConnectedCallback(false));
+  private void initTransport() throws Exception {
+    initTransport(3, new ConnectingCallback(false));
+    connectedFuture.get();
   }
 
-  private void initTransport(int startId, ConnectedCallback connectedCallback) {
-    this.connectedCallback = connectedCallback;
+  private void initTransport(int startId, ConnectingCallback connectingCallback) {
```

Comment:
Since this behaves differently than the other `initTransport()`, could it be renamed? Renaming the other `initTransport()` would be nicer in some ways, but there are so many usages of it renaming this one seems reasonable.